lua/acf/core/classes/simple.lua

```diff
@@ -29,6 +34,9 @@ function Classes.AddSimple(ID, Destiny, Data)
 	return Class
 end
 
+--- Indexes the simple classes stored in Entries into a new Namespace, with helper functions
+--- @param Namespace table The namespace to store the simple classes under
```

Comment:
Same as the grouped ones, the namespace given to this function will receive a new set of functions